rep-0149.rst

```diff
@@ -0,0 +1,1090 @@
+
+
+<maintainer> (multiple, but at least one)
+-----------------------------------------
+
+The name of the person maintaining the package.  All packages require
+a maintainer.  For orphaned packages see below.
+
+Attributes
+''''''''''
+
+ ``email="name@domain.tld"`` *(required)*
+
+  Email address of the maintainer.
+
+An orphaned package is one with no current maintainer.
+Orphaned packages should use the following maintainer information to
+guide volunteers how they can claim maintainership:
+
+Example
+'''''''
+
+::
+
+  <maintainer email="ros-orphaned-packages@googlegroups.com">Unmaintained see http://wiki.ros.org/MaintenanceGuide#Claiming_Maintainership</maintainer>
+
+
+<license> (multiple, but at least one)
+--------------------------------------
+
+Name of license for this package, e.g. BSD, GPL, LGPL.  In order to
+assist machine readability, only include the license name in this tag.
+For multiple licenses multiple separate tags must be used.  A package
+will have multiple licenses if different source files have different
+licenses.  Every license occurring in the source files should have
+a corresponding ``<license>`` tag.  For any explanatory text about
+licensing caveats, please use the ``<description>`` tag.
+
+Most common open-source licenses are described on the
+`OSI website <http://www.opensource.org/licenses/alphabetical>`_.
+
+Commonly used license strings:
+
+ - Apache 2.0
+ - BSD
+ - Boost Software License
+ - GPLv2
+ - GPLv3
+ - LGPLv2.1
+ - LGPLv3
+ - MIT
+ - Mozilla Public License Version 1.1
+
+<url> (multiple)
+----------------
+
+A Uniform Resource Locator for the package's website, bug tracker or
+source repository.
+
+It is a good idea to include ``<url>`` tags pointing users to these
+resources.  The website is commonly a wiki page on ``ros.org`` where
+users can find and update information about the package.
+
+Attributes
+''''''''''
+
+ ``type="TYPE"`` *(optional)*
+
+ The type should be one of the following identifiers: ``website``
+ (default), ``bugtracker`` or ``repository``.
+
+
+<author> (multiple)
+-------------------
+
+The name of a person who is an author of the package, as
+acknowledgement of their work and for questions.
+
+Attributes
+''''''''''
+
+ ``email="name@domain.tld"`` *(optional)*
+
+  Email address of author.
+
+
+Dependency tags
+---------------
+
+<build_depend> (multiple)
+'''''''''''''''''''''''''
+
+Declares a rosdep key or ROS package name that this package requires
+at build-time.  For system packages, the rosdep key will normally
+specify the "development" package, which frequently ends in ``"-dev"``.
+
+The ``build`` and ``buildtool`` dependencies are used to determine
+the build order of multiple packages.
+
+Attributes
+..........
+
+ All dependencies and relationships may restrict their applicability
+ to particular versions.  For each comparison operator an attribute
+ can be used.  Two of these attributes can be used together to
+ describe a version range.
+
```

Comment:
It might come handy to mention that rosdep currently ignores the `version_*` tags. Or add a link to issue https://github.com/ros-infrastructure/rosdep/issues/325 . Again, an example use-case might help. Here, we could mention that Debian package version constraints can be generated from these tags.

Reply:
This part as not changed since the previous REP. Therefore I won't focus on this in this REP but concentrate on the newly added attributes and tags. Please consider providing a PR to amend the existing REPs to add some example use case.